build/linux_aio_perfcheck.sh

```diff
@@ -3,8 +3,71 @@
 # Linux All-in-one Performance Collector 
 # Description:  shell script which collects performance data for analysis
 # About: https://github.com/samatild/LinuxAiOPerf
-# version: 1.21
-# Date: 10/Nov/2023
+# version: 1.30
+# Date: 04/Mar/2024
+     
+
+setLocalInstructions(){
+    clear
+    echo -e "\e[1;33m"
+    cat << "EOF"
+ ==================================================================
+        Instructions on how to update local Time Zone settings 
+ ==================================================================
+EOF
+
+    echo -e "\n\e[0;31m[Linux Generic procedure is]\e[0m"
+    echo -e "\033[1;34m# Check what Language packages we have installed\e[0m"
+    echo -e "locale -a\n"
+    echo -e "\033[1;34m# Install if necesarry\e[0m"
+    echo -e "localedef -i en_US -f UTF-8 en_US.UTF-8\n"
+    echo -e "\033[1;34m# Change LC_TIME to en_US.UTF-8\e[0m"
+    echo -e "localectl set-locale LC_TIME=en_US.UTF-8\n"
+    echo -e "\033[1;34m# Exit your current user session, and re-login\e[0m"
+    echo -e "\033[1;34m# We need to source new locale settings\e[0m\n"
+    echo -e "\033[1;34m# You can revert using same command\e[0m"
+    echo -e "localectl set-locale LC_TIME=<original value>\n"
+    
+    # Exit
+    echo "Once you are done with Time Zone adjustments, re-run the Collector script."
+    echo "Press any key to exit..."
+    read -s -n 1
+    exit 0 
+}
+
+
+localeValidation() {
+
+    CURRENT_LC_TIME=$(locale | grep LC_TIME | cut -d= -f2 | tr -d '"')
+    
+    if [[ "$CURRENT_LC_TIME" != "en_US.UTF-8" && "$CURRENT_LC_TIME" != "C.UTF-8" && "$CURRENT_LC_TIME" != "POSIX" ]]; then
```

Comment:
We compare with valid types : en_US.UTF-8 ; C.UTF-8 ; POSIX ; 
If compatible we continue, otherwise we prompt the user to change it manually.